python/langsmith/_internal/_operations.py

```diff
@@ -271,3 +273,36 @@ def serialized_run_operation_to_multipart_parts_and_context(
         acc_parts,
         f"trace={op.trace_id},id={op.id}",
     )
+
+
+def compress_multipart_parts_and_context(
```

Comment:
I'm encoding the multipart form manually here instead of using the request toolkit so I can add a final boundary when I drain the queue vs adding a final boundary in between each run.

Reply:
open to other suggestions as wlel

Reply:
I think this makes sense for now